mentors/clojure/t.json

```diff
@@ -6,5 +6,13 @@
         "link_url": null,
         "avatar_url": null,
         "bio": "I love Lisp, and functional programming. I've written a few apps in Clojure, and some with heavy Java interop."
+  },
+  {
+        "github_username": "tvirolai",
+        "name": "Tuomo Virolainen",
+        "link_text": null,
+        "link_url": null,
+        "avatar_url": null,
+        "bio": "Functional programmer and lisper. I got interested in Clojure in about 2015 and it quickly became my tool of choice for most tasks."
```

Comment:
These lines should be indented 2 spaces more than `{` and `}`.

Reply:
Fixed now.